### DISCUSSION TITLE
Public site

### DISCUSSION THREAD

**Discussion by Mixnetwork** [AzuraCast/AzuraCast#5751] [Ideas]
I use Azuracast for my Private station.
Now I want to post last played songs on my website, but i do not want that the Public Site or WEBDJ, Poscast Or scedule are visible on the net.
This is because otherwise the stream would become public, and I have to pay for the music I stream in Germany they call it GEMA and GVL, the cost here are too much for private use (about 500 Euro a year)

SO I want only to publish last played songs in a widget on a Wordpress site, is this possible without activating Public sites?

If not, perhaps It's possible to make an option, where you can choose which site can be public and which not for next update 👍

**Comment by Mozitux** [comment]
Hi, you can use now playing Azuracast API in a PHP file to do this.

**Reply by Mixnetwork** [reply]
OK, Thank you, I will try this

**Comment by Mixnetwork** [comment]
pfff too complicated for me, but anyway...thanks

**Reply by Mozitux** [reply]
You will find the beginning of an answer here:
https://github.com/AzuraCast/AzuraCast/discussions/5503
